src/utils/get-page-schema.js

```diff
@@ -0,0 +1,51 @@
+export const getPageSchema = (url, articleInfo = null) => {
```

Comment:
Does this also need to be added to `/`, `/learn`, `/community` as well?

Reply:
Yea I think we should add these to those pages as well.

Reply:
Will get confirmation, but I believe the abbreviated version, yes